setup.py

```diff
@@ -1,8 +1,13 @@
+from os import sys, path
+
 from skbuild import setup
 
+sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
+from itkVersion import VERSION
```

Comment:
>  could add it to py_modules if there is a need, 

That will be required, otherwise the file `itkVersion.py` wouldn't be package. (using the compliant and updated scikit-build)